scripts/mlrun.sh

```diff
@@ -1,9 +1,16 @@
 DOCKER_SERVER="${DOCKER_SERVER:-}"
-DOCKER_USERNAME="${DOCKER_USERNAME:-}"
-DOCKER_PASSWORD="${DOCKER_PASSWORD:-}"
+DOCKER_USERNAME="${DOCKER_USERNAME:- }"
+DOCKER_PASSWORD="${DOCKER_PASSWORD:- }"
 DOCKER_REGISTRY_URL="${DOCKER_REGISTRY_URL:-}"
-
-# Remove argo workflows CRDs and patch nemo's workflow controller to use namespaced mode:
+MLRUN_NAMESPACE="mlrun"
+# Set up a local docker registry on the minikube cluster only if docker registry is not provided:
+if [ -z "$DOCKER_REGISTRY_URL" ]; then
+  LOCAL_DOCKER_REGISTRY_FLAG=1
+else
+  LOCAL_DOCKER_REGISTRY_FLAG=0
+fi
```

Comment:
is it documented anywhere? I saw you deleted the env var and explanation stuff, perhaps you can keep them as optional?